model/src/main/kotlin/OrtResult.kt

```diff
@@ -405,7 +406,17 @@ data class OrtResult(
 
     private fun getPackages(): Set<CuratedPackage> = analyzer?.result?.packages ?: emptySet()
 
-    private fun getProjects(): Set<Project> = analyzer?.result?.projects ?: emptySet()
+    /**
+     * Return all [Project]s contained in this [OrtResult].
+     */
+    @JsonIgnore
+    fun getProjects(): Set<Project> = analyzer?.result?.projects ?: emptySet()
+
+    /**
+     * Return all [RuleViolation]s contained in this [OrtResult].
+     */
+    @JsonIgnore(true)
+    fun getRuleViolations(): List<RuleViolation> = evaluator?.violations ?: emptyList()
```

Comment:
Should we make this a `val` property? Prefer `.orEmpty()` instead of the `?:` fallback.

Reply:
Done `.orEmpty()`. Kept the fun as I slightly prefer it and for local consistency.